### PR TITLE
Feature/pick excisting theme

### PR DESCRIPTION
**Pull request by Richard87**
Solves #19

### CODE REVIEW

**Comment by mr-islam** [comment]
Sorry I'm late 😅! 

I just took a look and the changes seem great, improving a lot of code and the themes of course

I'll take a closer look and test over the next week or so 👍

Thank you again

**Comment by Richard87** [comment]
No worries, take your time, your extension already works great :D

File: settings/settings.html [hunk 1]
```diff
@@ -27,27 +27,36 @@ <h1>Settings</h1>
         </select>
 
         <br>
-        <strong>Tab line color</strong> from Solarized theme:
+        <em>light</em> theme: <select id="lightTheme">
+          <option value="Solarized">Built-in Solarized theme</option>
+          <option value="firefox-compact-light@mozilla.org">Firefox Light</option>
+        </select><br>
+        <em>dark</em> theme accent: <select id="darkTheme">
```
Comment (mr-islam) [inline settings/settings.html:34]: I think it should be just "dark theme", right? (Without "accent")

**Review by mr-islam — APPROVED** [review]
Okay, I've tested the updates and read through the code – it all looks great and it's awesome to see all the improvements to the code base specially. Thank you so much for helping to improve Zenfox!

File: settings/settings.html (hunk 1 above)
Comment (mr-islam) [reply settings/settings.html:34]: It's such a small thing I just made the change and am merging now ;)

**Pull request merged**